packages/modular-scripts/src/buildPackage.ts

```diff
@@ -400,8 +412,10 @@ async function makeBundle(
   }
 
   if (Object.keys(localImports).length > 0) {
-    console.log('Adding dependencies to the generated package.json:');
-    console.log(localImports);
+    logger.log('Adding dependencies to the generated package.json:');
+    Object.entries(localImports).forEach(([packageName, packageVersion]) => {
+      logger.log(`\t${packageName}: ${chalk.green(packageVersion)}`);
```

Comment:
I found that the indentation was irregular across things - so I though this was slightly tidier. Since `logger` outputs the package location for each line this means you can always see where a log line came from.